readme.md

```diff
@@ -71,6 +77,13 @@ Default: `false`
 
 Uppercase the first character: `foo-bar` → `FooBar`
 
+##### locale
+
+Type: `string | string[]`\
+Default: `undefined`
```

Comment:
I assume the default is the user's locale? If so, that should be the default here.

Reply:
Yes, updated as suggested.